modules/tinymce/src/core/main/ts/DragDropOverrides.ts

```diff
@@ -121,6 +131,54 @@ const moveGhost = (
 
   ghostElm.style.width = (width - overflowX) + 'px';
   ghostElm.style.height = (height - overflowY) + 'px';
+
+  // Code needed for dragging CEF elements (specifically fixing TINY-8874)
+  // The idea behind the algorithm is that the user will start dragging the
+  // CEF element to the edge of the editor and that would cause scrolling.
+  // The way that happens is that the user will trigger a mousedown event,
+  // then a mousemove event until they reach the edge of the editor. Then
+  // no event triggers. That's when I set an interval to keep scrolling the editor.
+  // Once a new event triggers I clear the existing interval and set it back to none.
+
+  const clientHeight = contentAreaContainer.clientHeight;
+
+  // current scroll level
+  const currentTop = win.scrollY;
+
+  state.on((state) => {
+    if (state.dragging) {
+      // This basically means that the mouse is close to the bottom edge
+      // (within MouseRange pixels of the bottom edge)
+      if (mouseY + MouseRange >= clientHeight) {
```

Comment:
This implementation doesn't support scrolling to the side, is this intentional? Is that solved elsewhere?

Reply:
Because I believe the editor is not scrollable to the sides so it supporting scrolling to the side wouldn't have benefits and would be hard (maybe impossible ?) to test.

Reply:
It's very easy to scroll sideways in the editor, I did check to make sure. Unless this is referring to something else? You can make a `div` with very wide width, and that needs to scroll?

Reply:
You are correct, I forgot about that. The way I tested was using a very large sentence and it was broken down multiple line instead of being one large line that can be scrolled sideways. I will add support for scrolling to the side 👍

Reply:
Done!